pkg/handlers/update_checker_spec.go

```diff
@@ -35,6 +44,91 @@ func (h *Handler) ConfigureAutomaticUpdates(w http.ResponseWriter, r *http.Reque
 		return
 	}
 
+	clientSet, err := k8sutil.GetClientset()
+	if err != nil {
+		logger.Error(err)
+		updateCheckerSpecResponse.Error = "failed to create k8s client"
+		JSON(w, 500, updateCheckerSpecResponse)
+		return
+	}
+
+	selectorLabels := map[string]string{
+		"metadata.name": "kotsadm-confg",
+	}
+	listOpts := metav1.ListOptions{
+		FieldSelector: labels.SelectorFromSet(selectorLabels).String(),
+	}
+
+	configMaps, err := clientSet.CoreV1().ConfigMaps(util.PodNamespace).List(context.TODO(), listOpts)
+	if err != nil && !kuberneteserrors.IsNotFound(err) {
```

Comment:
List doesn't return "not found" errors.  Use `Get` instead of `List` like here https://github.com/replicatedhq/kots/blob/main/pkg/kotsadm/main.go#L1084

Reply:
ah thanks, done